TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/ResultsToLogsConsumerTest.java

```diff
@@ -32,6 +33,10 @@
 @Slf4j
 @WrapWithNettyLeakDetection(repetitions = 4)
 class ResultsToLogsConsumerTest extends InstrumentationTest {
+    static {
+        // Synchronize logging to for assertions
+        LogManager.setFactory(new Log4jContextFactory(new ClassLoaderContextSelector()));
```

Comment:
how does the code achieve the comment?

Reply:
The ClassLoaderContextSelector forces a unified synchronous logger